src/test/resources/features/refapp-2.x/06-user/users.feature

```diff
@@ -1 +1,30 @@
-Feature: Users Management
\ No newline at end of file
+Feature: User Account Management
+
+  Background:
+    Given a user clicks on system administartion app from home page
```

Comment:
Each feature file should correspond to one step definitions, where are these step definitions corresponding to background user stories

Reply:
@sherrif10 could you give more light on the question, I did  not get the question quite well !

Reply:
@sherrif10  I did not get your question well ?